frontend/src/main/scala/akka/viz/frontend/FrontendApp.scala

```diff
@@ -123,38 +127,69 @@ object FrontendApp extends JSApp with Persistence
   val messagesPanel = new MessagesPanel(selectedActors)
   val userIsEnabled = Var(false)
   val onOffPanel = new OnOffPanel(_eventsEnabled, userIsEnabled)
+  val connectionAlert = new Alert()
 
   @JSExport("toggleActor")
   def toggleActor(name: String) = actorSelector.toggleActor(name)
 
-  def main(): Unit = {
-
-    val upstream = ApiConnection(
-      webSocketUrl("stream"),
-      handleDownstream(messagesPanel.messageReceived)
-    ) // fixme when this will need more callbacks?
-
-    selectedMessages.triggerLater {
-      console.log(s"Will send allowedClasses: ${selectedMessages.now.mkString("[", ",", "]")}")
-      import upickle.default._
-      upstream.send(write(SetAllowedMessages(selectedMessages.now)))
-    }
+  val maxRetries = 10
 
-    selectedActors.triggerLater {
-      console.log(s"Will send ObserveActors: ${selectedActors.now.mkString("[", ",", "]")}")
-      import upickle.default._
-      upstream.send(write(ObserveActors(selectedActors.now)))
-    }
+  def main(): Unit = {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Method is longer than 50 lines](https://www.codacy.com/app/lustefaniak/akka-viz/file/2051036535/issues/source?bid=3121282&fileBranchId=3133751#l137)